crates/bench_utils/src/lib.rs

```diff
@@ -0,0 +1,188 @@
+mod utils;
```

Comment:
I would add a doc comment (`//!`) explaining the file, e.g. "these are the macros used to generate programs for the flowistry benchmark" or something.